.github/PULL_REQUEST_TEMPLATE/feature.md

```diff
@@ -0,0 +1,27 @@
+* **Please check if the PR fulfills these requirements**
+
+- [ ] E2E Tests for the changes have been added (for bug fixes / features) via Cypress
+- [ ] Docs have been added / updated (for bug fixes / features)
```

Comment:
What about unit tests with RSpec? I think they should be included as well.

Reply:
Oh i forgot them